src/transformers/modeling_tf_utils.py

```diff
@@ -219,7 +219,7 @@ def hf_compute_loss(self, labels, logits):
         # make sure only labels that are not equal to -100
         # are taken into account as loss
         if tf.math.reduce_any(labels == -1):
-            warnings.warn("Using `-1` to mask the loss for the token is deprecated. Please use `-100` instead.")
+            tf.print("Using `-1` to mask the loss for the token is deprecated. Please use `-100` instead.")
```

Comment:
Why not use our `logger` with `logger.warn`?

Reply:
Ah, I didn't realize our logger was a TF logger that wrote graph ops! Let me try that.

Reply:
Update: Turns out our TF logger doesn't emit graph ops, so the warnings start firing again when I try that:
![image](https://user-images.githubusercontent.com/12866554/151836929-95b1456f-2106-4666-8091-2cce39fc3f81.png)

Reply:
Ok to use `tf.print` then.

Reply:
Cool!